### PR TITLE
reduce unnecessary logs for mainnet

### PR DESCRIPTION
**Pull request by dayuanc**
https://github.com/iotexproject/iotex-core/issues/2794

Remove unnecessary logs which just used for printing values purpose for mainnet.

### CODE REVIEW

**Comment by raullenchai** [comment]
I think these debug logs are fine - as they are not ingested by our ELK - we need to downgrade the unnecessary INFO/WARNING/ERROR logs

**Comment by dustinxie** [comment]
i think the request is to review the current logs, and check if any`WARN/INFO` can reduce to `DEBUG`

**Comment by dayuanc** [comment]
So from kibana 90% logs come from error level and most of produced by this line: https://github.com/iotexproject/iotex-core/blob/0ffe3834aa73af63975b36e87a01a3a4967b8f9a/action/protocol/execution/evm/evmstatedbadapter.go#L682
and the left 8% logs from https://github.com/iotexproject/iotex-core/blob/0ffe3834aa73af63975b36e87a01a3a4967b8f9a/server/itx/heartbeat.go#L100 and https://github.com/iotexproject/iotex-core/blob/0ffe3834aa73af63975b36e87a01a3a4967b8f9a/server/itx/heartbeat.go#L146

**Comment by raullenchai** [comment]
> So from kibana 90% logs come from error level and most of produced by this line:
> 
> https://github.com/iotexproject/iotex-core/blob/0ffe3834aa73af63975b36e87a01a3a4967b8f9a/action/protocol/execution/evm/evmstatedbadapter.go#L682
> 
> 
> and the left 8% logs from
> https://github.com/iotexproject/iotex-core/blob/0ffe3834aa73af63975b36e87a01a3a4967b8f9a/server/itx/heartbeat.go#L100
> 
> and
> https://github.com/iotexproject/iotex-core/blob/0ffe3834aa73af63975b36e87a01a3a4967b8f9a/server/itx/heartbeat.go#L146
> 
> I think we can delete the `error` log and downgrade the info log. @raullenchai @dustinxie

Debug logs are disabled by default unless our eng enabled it for debugging

**Review by raullenchai — APPROVED** [review]

File: tools/minicluster/minicluster.go [hunk 1]
```diff
@@ -384,7 +384,7 @@ func main() {
 				return
 			}
 
-			log.S().Info("Fp token transfer test pass!")
+			log.S().Debug("Fp token transfer test pass!")
```
Comment (CoderZhi) [inline tools/minicluster/minicluster.go:387]: revert. minicluster is not running in product

File: tools/staterecoverer/staterecoverer.go [hunk 2]
```diff
@@ -94,7 +94,7 @@ func main() {
 	sf := svr.ChainService(cfg.Chain.ID).StateFactory()
 	dao := svr.ChainService(cfg.Chain.ID).BlockDAO()
 	if err := bc.Start(context.Background()); err == nil {
-		log.L().Info("State DB status is normal.")
+		log.L().Debug("State DB status is normal.")
```
Comment (CoderZhi) [inline tools/staterecoverer/staterecoverer.go:97]: same for this tool

File: action/protocol/execution/evm/evmstatedbadapter.go [hunk 3]
```diff
@@ -680,7 +680,6 @@ func (stateDB *StateDBAdapter) GetCommittedState(evmAddr common.Address, k commo
 	v, err := contract.GetCommittedState(hash.BytesToHash256(k[:]))
 	if err != nil {
 		log.L().Debug("Failed to get committed state.", zap.Error(err))
-		stateDB.logError(err)
```
Comment (CoderZhi) [inline action/protocol/execution/evm/evmstatedbadapter.go:683]: revert

Comment (dayuanc) [reply action/protocol/execution/evm/evmstatedbadapter.go:683]: fixed.

File: tools/minicluster/minicluster.go (hunk 1 above)
Comment (dayuanc) [reply tools/minicluster/minicluster.go:387]: fixed.

File: tools/staterecoverer/staterecoverer.go (hunk 2 above)
Comment (dayuanc) [reply tools/staterecoverer/staterecoverer.go:97]: fixed.

**Review by CoderZhi — APPROVED** [review]

**Pull request merged**